src/components/Metrics.tsx

```diff
@@ -14,24 +14,58 @@
  * limitations under the License.
  ***************************************************************************** */
 
+import { reaction } from 'mobx';
+import { useEffect, useMemo, useState } from 'react';
 import { createUseStyles } from 'react-jss';
+import { useBoxesStore } from '../hooks/useBoxesStore';
+import { useSchemaStore } from '../hooks/useSchemaStore';
+import { BoxEntity } from "../models/Box";
 
 const useStyles = createUseStyles({
 	container: {
 		border: '1px solid',
 		gridArea: 'metrics',
 		borderRadius: 6,
+		overflow: 'hidden'
 	},
+	metriks: {
+		height: '100%',
+		width: '100%',
+		border: 'none',
+	}
 });
 
 function Metrics() {
 	const classes = useStyles();
+	const schemaStore = useSchemaStore();
+	const boxesStore = useBoxesStore();
 
-	return (
+	const [componet, setComponet] = useState<string>('');
+
+	const source = useMemo(() => {
+		const namespace = schemaStore.selectedSchema;
+
+		return `http://th2-qa:30000/grafana/d-solo/logs/logs?orgId=1&refresh=10s&var-namespace=th2-${namespace}&var-component=${componet}&theme=light&panelId=8`;
+	}, [schemaStore.selectedSchema, componet]);
+
+
+	useEffect(() => {
+		const boxSubscription = reaction(
+			() => boxesStore.selectedBox,
+			box => {
+				setComponet(box?.name || '')
+			},
+		);
+
+		return boxSubscription;
+	}, [boxesStore.selectedBox]);
+
+	return boxesStore.selectedBox && (
```

Comment:
i think source should be nullable if no component selected and we should return based on source, not based on boxesStore.selectedBox